sipa/model/hss/user.py

```diff
@@ -271,7 +283,7 @@ def userdb(self):
 
     @property
     def has_connection(self):
-        return self._pg_account.properties.active
+        return True
```

Comment:
A static `has_connection = True` would've achieved the same, but I don't care about cosmetics atm.

Reply:
I agree with you, but _in theory_ the best practice would be to not delete accounts in Abe and mark the as closed in some sense. Then we would need a more sophisticated function here.

However, I've heard that Pycroft offers better solutions anyway ;)